test/mpi/threads/comm/idup_deadlock.c

```diff
@@ -41,6 +41,7 @@ MTEST_THREAD_RETURN_TYPE test_comm_dup(void *arg)
             wait = 0;
             for (i = 0; i < NUM_THREADS; i++)
                 wait += start_idup[i];
+            MTest_thread_yield();
```

Comment:
@shintaro-iwasaki `MTest_thread_yield()` is not in a header file that is included in `idup_deadlock.c`, causing a compiler warning. Could you fix it?